plugins/xkcd.py

```diff
@@ -13,7 +13,6 @@
 def xkcd_info(xkcd_id, url=False):
     """ takes an XKCD entry ID and returns a formatted string """
     request = requests.get("http://www.xkcd.com/" + xkcd_id + "/info.0.json")
-    request.raise_for_status()
```

Comment:
Why do you remove the error handling?

Reply:
I haven't touched this since October, when I made the minor URL change. I'm not sure what happened here, but I haven't committed anything in here in a while.